docs/openapi/components/schemas/common/TransformEvent.yml

```diff
@@ -0,0 +1,203 @@
+additionalProperties: false
+example: |-
+  {
+    "type":[
+      "Event",
+      "TransformEvent"
+    ],
+    "eventType":"Smelt",
+    "eventId":"30ee76a2-9ff9-4884-8ecf-c85c9f3343f4",
+    "eventTime":"2022-01-14T18:24:31Z",
+    "place":{
+      "geo":{
+        "type":[
+          "GeoCoordinates"
+        ],
+        "latitude":42.8864,
+        "longitude":-78.8784
+      },
+      "type":[
+        "Place"
+      ],
+      "address":{
+        "type":[
+          "PostalAddress"
+        ],
+        "postalCode":"",
+        "addressRegion":"New York",
+        "streetAddress":"",
+        "addressCountry":"USA",
+        "addressLocality":"Buffalo"
+      }
+    },
+    "actor":[
+      {
+        "name":"Fabricator Co.",
+        "type":[
+          "Organization"
+        ]
+      }
+    ],
+    "products":[
+      {
+        "id":"5c93de07-5d94-4357-85cf-aefc9ccbff97",
+        "name":"Carbon and alloy long product",
+        "type":[
+          "Product"
+        ],
+        "weight":{
+          "type":[
+            "MeasuredValue"
+          ],
+          "value":"3",
+          "unitCode":"Tonne"
+        },
+        "description":"Bars/rods, i/nas,hr, in irreg wnd coils of circ cross section, <14mm in dia",
+        "manufacturer":{
+          "name":"Fabricator Co.",
+          "type":[
+            "Organization"
+          ]
+        }
+      },
+      {
+        "id":"9d237753-b377-492a-a8c8-a28fe9f3457b",
+        "name":"Scrap Steel",
+        "type":[
+          "Product"
+        ],
+        "weight":{
+          "type":[
+            "MeasuredValue"
+          ],
+          "value":"10",
+          "unitCode":"Tonne"
+        },
+        "description":"Consumed product",
+        "manufacturer":{
+          "name":"Steel Co.",
+          "type":[
+            "Organization"
+          ]
+        }
+      },
+      {
+        "id":"adf9a0fe-a117-4728-a323-b941b5bcf011",
+        "name":"Iron Ore",
+        "type":[
+          "Product"
+        ],
+        "weight":{
+          "type":[
+            "MeasuredValue"
+          ],
+          "value":"12",
+          "unitCode":"Tonne"
+        },
+        "description":"Consumed product",
+        "manufacturer":{
+          "name":"Steel Co.",
+          "type":[
+            "Organization"
+          ]
+        }
+      },
+      {
+        "id":"c72a3cb7-3d2f-4d42-b398-2deb6e6ace66",
+        "name":"Scrap Steel",
+        "type":[
+          "Product"
+        ],
+        "weight":{
+          "type":[
+            "MeasuredValue"
+          ],
+          "value":"10",
+          "unitCode":"Tonne"
+        },
+        "description":"Consumed product",
+        "manufacturer":{
+          "name":"Steel Co.",
+          "type":[
+            "Organization"
+          ]
+        }
+      }
+    ],
+    "newProducts":[
+      "https://api-staging.fabricator.metaltrail.com/v1/credentials/verify/5c93de07-5d94-4357-85cf-aefc9ccbff97"
+    ],
+    "consumedProducts":[
```

Comment:
might be good to chat a bit about these URLS

Reply:
updated